package.json

```diff
@@ -46,22 +46,22 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@capacitor/android": "^3.0.0",
-    "@capacitor/cli": "^3.0.0",
-    "@capacitor/core": "^3.0.0",
+    "@capacitor/android": "^4.0.0",
+    "@capacitor/cli": "^4.0.0",
+    "@capacitor/core": "^4.0.0",
     "@capacitor/docgen": "^0.0.18",
-    "@capacitor/ios": "^3.0.0",
+    "@capacitor/ios": "^4.0.0",
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/prettier-config": "^1.0.1",
     "@ionic/swiftlint-config": "^1.1.2",
     "@types/mixpanel-browser": "^2.38.0",
     "eslint": "^7.11.0",
-    "prettier": "~2.2.0",
-    "prettier-plugin-java": "~1.0.0",
+    "prettier": "~2.3.0",
+    "prettier-plugin-java": "~1.0.2",
     "rimraf": "^3.0.2",
     "rollup": "^2.32.0",
     "swiftlint": "^1.0.1",
-    "typescript": "~4.0.3"
+    "typescript": "~4.1.5"
   },
   "peerDependencies": {
     "@capacitor/core": "^3.0.0"
```

Comment:
This needs to be updated to support V4
`npm ERR! Conflicting peer dependency: @capacitor/core@3.8.0
npm ERR! node_modules/@capacitor/core
npm ERR!   peer @capacitor/core@"^3.0.0" from @houseninja/capacitor-mixpanel@1.0.0-beta.1
npm ERR!   node_modules/@houseninja/capacitor-mixpanel
npm ERR!     @houseninja/capacitor-mixpanel@"^1.0.0-beta.1" from the root project`